pkg/common/utils/chunk.go

```diff
@@ -57,6 +57,7 @@ func ParseChunkSize(r network.Reader) (int, error) {
 	if err != nil {
 		return -1, errors.NewPublic(fmt.Sprintf("cannot read '\n' char at the end of chunk size: %s", err))
 	}
+	// todo support chunk extension
```

Comment:
What is chunk extension?

Reply:
can refer to this issue https://github.com/cloudwego/hertz/issues/334